conf/messages.en

```diff
@@ -501,7 +501,7 @@ registration.received.p5 = You will need to complete a return and pay the levy o
 registration.received.p6 = You will not receive an email or notification from HMRC to remind you to submit a return.
 registration.received.p7 = The next deadline for submitting a return and making a payment is 30 September 2023. You will be charged interest if you make a late payment.
 registration.received.p8 = You will not receive an email or notification from HMRC to remind you to submit a return.
-registration.received.p9 = When you complete your yearly return, you will need to tell us about:
+registration.received.p8 = When you complete your yearly return, you will need to tell us about:
```

Comment:
Looks like we have two messages that share the p8 key. (503 & 504)

Should 504 be p9?

Reply:
One of the p8's is a duplicate so removed it.

Reply:
This is Ok.